tests/gui/runner.rs

```diff
@@ -26,12 +26,16 @@ fn get_available_browser_ui_test_version() -> Option<String> {
 }
 
 fn expected_browser_ui_test_version() -> String {
-    let content = read_to_string(".github/workflows/main.yml")
-        .expect("failed to read `.github/workflows/main.yml`");
-    for line in content.lines() {
-        let line = line.trim();
-        if let Some(version) = line.strip_prefix("BROWSER_UI_TEST_VERSION:") {
-            return version.trim().replace('\'', "");
+    let content = read_to_string("package.json").expect("failed to read `package.json`");
+    let json = json::parse(&content).expect("invalid JSON");
```

Comment:
Can this use serde_json since it is already in the dependency tree?

Reply:
Good idea.